corehq/apps/es/fake/groups_fake.py

```diff
@@ -0,0 +1,11 @@
+from corehq.apps.es.fake.es_query_fake import HQESQueryFake
+
+
+class GroupESFake(HQESQueryFake):
+    _all_docs = []
+
+    def domain(self, domain):
+        return self._filtered(lambda doc: doc['domain'] == domain)
```

Comment:
You could probably put this (and the analogs to any `filters.py` filters) in `HQESQueryFake` - as the implementation is the same for everything except `UserESFake`